action_server/src/robocorp/action_server/_server.py

```diff
@@ -243,11 +244,16 @@ def expose_later(loop):
 
     def _on_started_message(self, **kwargs):
         (host, port) = _get_currrent_host()
-        log.info(f"\n  ⚡️ Action Server started at http://{settings.address}:{port}")
+
+        log.info(
+            colored("\n  ⚡️ Action Server started at: ", "green", attrs=["bold"])
+            + colored(f"http://{settings.address}:{port}", "light_blue")
+        )
 
         if api_key:
             log.info(
-                f'  🔑 API Authorization key: {{ "Authorization": "Bearer {api_key}"}}'
+                colored("  🔑 API Authorization key: ", attrs=["bold"])
```

Comment:
Question on: 
`API Authorization key: { "Authorization": "Bearer Kt06mKrycBAYWtZ8QwiM_EfBL4O9CWbSKWuTkow0Z98"}`
..where do we need this in the raw format?

If most of the GPT config are implemented in a way that user selects auth. type from a drop-box and copies in "API Key", then should we parse and present these in that from?

Reply:
Valid point, updated it to be just:
<img width="729" alt="image" src="https://github.com/robocorp/robocorp/assets/31288628/da3a39b0-d42e-4ce3-b8ba-880a08b1d686">